python/python/bystro/search/utils/messages.py

```diff
@@ -22,6 +27,32 @@ class IndexJobCompleteMessage(CompletedJobMessage, frozen=True, kw_only=True):
     results: IndexJobResults
 
 
+class BinomialMafFilter(
+    Struct, frozen=True, tag="binomMaf", tag_field="key", forbid_unknown_fields=True
```

Comment:
What are the tradeoffs of this approach to tagging vs `tag=True`?  The latter approach would add a key-value pair `"type":"BinomialMafFilter"`to the json, which (at least imo) seems easier to parse as a piece of metadata indicating what kind of Struct the json represents.  This approach would add `"key":"binomMaf"` which seems a little more open to interpretation, wdyt?

Reply:
`tag=True` would require our webapp frontend to be dependent on what we name this class. `key: "binomMaf"` is what the frontend sends (and has for years), and what SeqFromQuery.pm expected.